File: Data Structures & Algorithms/duplicate-integer/submission-3.java
public class Solution {
    public boolean hasDuplicate(int[] nums) {
        if (nums.length == 0) {
            return false;
        }

        int point1 = 0;
        int point2 = 1;

        while (point1 < nums.length - 1) {
            if (point2 >= nums.length) {
                point1++;
                point2 = point1 + 1;
                continue;
            }

            if (nums[point1] == nums[point2]) {
                return true;
            }

            point2++;
        }

        return false;
    }
}

File: Data Structures & Algorithms/two-integer-sum-ii/submission-3.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        

    
        lenList = len(nums)

        l = 0
        r = lenList - 1

        while l< r:
            if nums[l]+nums[r] == target:
                return [l+1,r+1]
            
            if nums[l]+ nums[r]> target:
                r-=1
            elif nums[l] + nums[r] < target:
                l+=1
        
         